src/Components/src/Microsoft.AspNetCore.Components/Rendering/Renderer.cs

```diff
@@ -77,8 +85,111 @@ protected void RenderRootComponent(int componentId)
         /// <param name="initialParameters">The <see cref="ParameterCollection"/>with the initial parameters to use for rendering.</param>
         protected void RenderRootComponent(int componentId, ParameterCollection initialParameters)
         {
+            ContinueAfterLifecycleTask(RenderRootComponentAsync(componentId, initialParameters));
+        }
+
+        private async void ContinueAfterLifecycleTask(Task task)
```

Comment:
The naming of this threw me (and the comment inside it about issuing a further render). It's not really a lifecycle task, nor are we continuing after it, and we'd never issue a further render.

Would it be more accurate to call this `ReportAsyncExceptions`?

Reply:
TBH I'd also be OK with dropping the synchronous overloads of `RenderRootComponent` and only having `RenderRootComponentAsync`. Don't mind whether we actually do that right now though.

Reply:
I'll file an issue for it

Reply:
https://github.com/aspnet/AspNetCore/issues/6794